package.json

```diff
@@ -25,6 +26,7 @@
   "scripts": {
     "start": "while(true) do node standupbot.js; sleep 5; done",
     "test": "mocha",
+    "migrations": "knex migrate:latest",
     "lint": "eslint ."
   },
   "engine": "node >= 0.8.14"
```

Comment:
Do you think we should update the engine requirements and start using ES6? That doesn't belong in this PR, but it would be nice to use the type system to make sure `STATUS` is a `const` etc.